sw/otbn/code-snippets/p256.s

```diff
@@ -1801,6 +1801,12 @@ dptr_y:
 dptr_d:
   .zero 4
 
+/* pointer to verification result x_r aka x_1 (dptr_x_r) */
+.globl dptr_x_r
+.balign 4
+dptr_x_r:
+  .zero 4
+
```

Comment:
The issue I see here is that we have now an index table with 9 entries. Originally the plan was here to have a binary that always has the index table at dmem address 0 and occupies exactly one 256 bit cell. Therefore `dptr_rnd` was double booked since the verification routine never needs a `rnd` value (and the ones that use `rnd` don't need a verification result)

Reply:
Is there a reason to only use a single 256b word for the vector table? Yes, the new alignment will "waste" 256-32b of DMEM, but we're not even close to memory pressure.

Reply:
It might get more critical with other curves where we need to provide domain parameters, but you're right, we probably won't use all 128 cells there either. Having all the pointers in the 1st cell just "looked" a bit nicer to me, that's all. Not really emotional about it. We can change it.

Reply:
Let's avoid optimizations until we need them for the sake of clarity. Using the same field for two very different purposes caused enough confusion already, so I'd prefer to just pay the price for a bit more DMEM usage, until we actually run into a DMEM pressure situation.